src/theia/extension.theiaChrome.ts

```diff
@@ -0,0 +1,235 @@
+/*
+* This program and the accompanying materials are made available under the terms of the *
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at *
+* https://www.eclipse.org/legal/epl-v20.html                                      *
+*                                                                                 *
+* SPDX-License-Identifier: EPL-2.0                                                *
+*                                                                                 *
+* Copyright Contributors to the Zowe Project.                                     *
+*                                                                                 *
+*/
+
+import { Builder, By, Key, until, Button } from "selenium-webdriver";
+// tslint:disable-next-line: no-submodule-imports
+import * as chrome from "selenium-webdriver/chrome";
+import { TheiaLocator, DatasetsLocators, UssLocators, JobsLocators, TheiaNotificationMessages } from "./Locators";
+
+const WAITTIME = 30000;
+const SLEEP = 2000;
```

Comment:
Can we have the same name for this 2 second sleep in both Chrome and Firefox test files?

Reply:
Gave same variable name in all the test files